### ISSUE TITLE
Legal License Change Issues

### ISSUE THREAD

**Issue by leviem1** [insidersec/insider#37]
I noticed you changed how your software was licensed between a couple of versions. Version 1.0.0 was released under LGPL-3.0, version 2.0.0 was released under no license (automatic copyright), and version 2.0.5 was released under GPL-3.0.

I'm assuming that this wasn't intentional as this has some nasty legal implications that render this project basically unusable until they are resolved. I guess the options would be to...
1. Revert to the LGPL-3.0 license along with any code not made under that license (2.0.0 - 2.0.4)
2. Change to GPL-3.0 and provide the written approvals from each contributor, removing code from any contributor that denies permission (approvals would ideally be accessible from the repository as outlined [here](https://opensource.stackexchange.com/a/2032))
3. Revert to the LGPL-3.0 license and provide the written approvals from each contributor who made contributions since 1.0.1 to license under LGPL-3.0 as explained above.

The root of the issue is that the developers who contributed to version 1.0.0 - 1.0.1 have not given permission to have the license for their code removed as it was in version 2.0.0 - 2.0.4, and developers who contributed to 2.0.0 - 2.0.4 have not given permission to license their code under GPL-3.0 as in 2.0.5. There's no way to prove that proper permission was granted by contributors since version 2.0.0. This may seem nit-picky, but the issue can result in DMCA takedowns which threaten the usability of the software. People using code from version 2.0.0 - 2.0.5 of this project may also be subject to legal consequences.

There's probably a loophole somewhere in the fact that your README hasn't been updated which you might be able to use as "proof" that the licence change was unintentional. However, I'm not qualified to give legal advice, so I suggest you speak to a lawyer before moving forward.

TL;DR call a lawyer.

**Comment by omatron** [comment]
Hello @leviem1 thanks for the heads up, we are working on it.

**Comment by danilodeveloper** [comment]
working on #38 
Tks @leviem1!

**Comment by omatron** [comment]
New License is up https://github.com/insidersec/insider/blob/master/LICENSE commit: https://github.com/insidersec/insider/commit/fe5d2febc6cfa8138dc6507b5c0ccbb68f971dd4 

Thanks @leviem1 !

**Comment by omatron** [comment]
New License is up https://github.com/insidersec/insider/blob/master/LICENSE commit: https://github.com/insidersec/insider/commit/fe5d2febc6cfa8138dc6507b5c0ccbb68f971dd4 

Thanks @leviem1 !
